package.json

```diff
@@ -71,6 +74,8 @@
     "istanbul": "0.4.3",
     "mocha": "2.4.5",
     "npm-run-all": "2.1.1",
+    "postcss-cli": "2.6.0",
+    "postcss-import": "8.1.2",
```

Comment:
We don't have any `@import` rules in our CSS, perhaps we can drop this for now and add it when we need it?